### ISSUE TITLE
Red images in ULAP results

### ISSUE THREAD

**Issue by GioFic95** [wangyanckxx/Single-Underwater-Image-Enhancement-and-Color-Restoration#4]
Hello, I tried to run your implementation of the ULAP method, but most of the resulting images are completely red.

In the attached images you can see original images from [TrashCan dataset](https://conservancy.umn.edu/handle/11299/214865) and resulting images produced with ULAP.

Since in your paper you write "The best restoration images are produced by IBLA and ULAP", I suppose there is something wrong with my output. Other methods such as IBLA seems to work correctly with my input images. Do you have any suggestions on how to solve this problem?
Thank you in advance.

Original images:
![image](https://user-images.githubusercontent.com/11423697/116489151-89798200-a894-11eb-89ff-f9233bd71b86.png)

ULAP results:
![image](https://user-images.githubusercontent.com/11423697/116489037-38698e00-a894-11eb-993d-3bde70bea012.png)

**Comment by stevenli11** [comment]
Hello, I would like to ask why there are no processed images in the Outputimages folder after my program runs. Is it that I created the wrong path?  
![image](https://user-images.githubusercontent.com/63144515/142967007-cd5e15ac-2171-43fa-a3da-5764902f888f.png)

**Comment by dddmmmyyy1998** [comment]
> Hello, I would like to ask why there are no processed images in the Outputimages folder after my program runs. Is it that I created the wrong path? ![image](https://user-images.githubusercontent.com/63144515/142967007-cd5e15ac-2171-43fa-a3da-5764902f888f.png)
你要在main函数中改一下存放的路径，cv.imwrite里面的参数要改

**Comment by HHUCKK** [comment]
> Hello, I tried to run your implementation of the ULAP method, but most of the resulting images are completely red.
> 
> In the attached images you can see original images from [TrashCan dataset](https://conservancy.umn.edu/handle/11299/214865) and resulting images produced with ULAP.
> 
> Since in your paper you write "The best restoration images are produced by IBLA and ULAP", I suppose there is something wrong with my output. Other methods such as IBLA seems to work correctly with my input images. Do you have any suggestions on how to solve this problem? Thank you in advance.
> 
> Original images: ![image](https://user-images.githubusercontent.com/11423697/116489151-89798200-a894-11eb-89ff-f9233bd71b86.png)
> 
> ULAP results: ![image](https://user-images.githubusercontent.com/11423697/116489037-38698e00-a894-11eb-993d-3bde70bea012.png)

Yes,I got the same problem as you when I was using GBdehazingRCorrection. Have you solved it? If yes, could you give me some advise like using which method?
